File: testsuite/octave-modular/util/check_accuracy.m
function y = check_accuracy(accuracy, train, test)
	printf("train: %e, test: %e <--- accuracy: %e\n", train, test, accuracy);

	if train>accuracy
		y=false;
	elseif test>accuracy
		y=false;
	else
		y=true;
	end

File: testsuite/r/util/check_accuracy.R
check_accuracy <- function(accuracy, type, data) {
	if (regexpr(type, 'kernel')) {
		print(paste('train:', data[1], ', test ', data[2], ' <--- accuracy ', accuracy))
	} else {
		print(paste('Unknown accuracy type', type, '!'))
		return(FALSE)
	}

	for (i in 1:length(data)) {
		if (data[i]>accuracy) {
			return(FALSE)
		}
	}

	return(TRUE)
}
